docs/advanced-usage/access-control/private-docker-registries/index.md

```diff
@@ -0,0 +1,9 @@
+# Private Docker registries
+
+If your workflow needs to access private Docker images, follow the docs depending on the Docker registry.
```

Comment:
If your workflow needs to access private Docker images, you will need to grant access to the REANA platform. The exact technique varies depending on the Docker registry. Please follow instructions below.